src/helpers/validator.ts

```diff
@@ -98,21 +98,20 @@ export const validateMessage = (
     }
 
     const typeInLowerCase = result.type?.toLowerCase();
-    if (
-      typeInLowerCase?.includes('_transaction') &&
-      typeInLowerCase !== result.type
-    ) {
-      return {
-        success: false,
-        error: "Value '<coin>_transaction' must be in lower case",
-      };
-    }
-
-    if (typeof result.amount !== 'string' || !isStringAmount(result.amount)) {
-      return {
-        success: false,
-        error: "Field 'amount' must be a string, representing a number",
-      };
+    if (typeInLowerCase?.includes('_transaction')) {
```

Comment:
You can use `.endsWith('_transaction')` or even better a RegExp.

Reply:
why is regexp better in your opinion?

Reply:
Because `random-string#123_transaction` will also pass the `if` statement, even though it is not a valid symbol.

Reply:
https://aips.adamant.im/AIPS/aip-12

> type represents token’s network and looks like `tickerSymbol_transaction`, e.g., `eth_transaction`. Ticker symbol must be in lower case, but apps must process it in any case for backwards compatibility. Mandatory.

@bludnic 
We can't be sure that tickers will not include `#` in future. E.g., tickers like `$CWIF` are usual these times.

Reply:
Then, we have to use `.endsWith('_transaction')` at least, because `.includes('_transaction')` will also match with "eth_transaction_random_string"